core/plugins/storage/ceph.py

```diff
@@ -514,6 +517,36 @@ def local_osds(self):
         self._local_osds = self._get_local_osds()
         return self._local_osds
 
+    def _get_pools(self, pools=None, crush_rule=-1):
```

Comment:
can we give this a more specific name like _filter_pools_by_rule() and also don't set pool=None since it always needs to be provided for this method to be useful.

Reply:
done